test_unstructured/partition/test_msg.py

```diff
@@ -156,102 +132,49 @@ def test_partition_msg_from_file_exclude_metadata():
         assert elements[i].metadata.to_dict() == {}
 
 
-def test_partition_msg_can_process_attachments(tmp_path: pathlib.Path):
-    file_path = example_doc_path("fake-email-attachment.msg")
-    tmp_dir_path = str(tmp_path)
-    extract_msg_attachment_info(filename=file_path, output_dir=tmp_dir_path)
-    attachment_filename = str(tmp_path / str(ATTACH_EXPECTED_OUTPUT[0]["filename"]))
-
-    mocked_last_modification_date = "2029-07-05T09:24:28"
-
-    attachment_elements = partition_text(
-        filename=attachment_filename,
-        metadata_filename=attachment_filename,
-        metadata_last_modified=mocked_last_modification_date,
-    )
-    expected_metadata = attachment_elements[0].metadata
-    expected_metadata.file_directory = None
-    expected_metadata.attached_to_filename = file_path
-
-    elements = partition_msg(
-        filename=file_path,
-        attachment_partitioner=partition_text,
-        process_attachments=True,
-        metadata_last_modified=mocked_last_modification_date,
-    )
-
-    # This test does not need to validate if hierarchy is working
-    # Patch to nullify parent_id
-    expected_metadata.parent_id = None
-    elements[-1].metadata.parent_id = None
-
-    assert elements[0].text.startswith("Hello!")
-    for element in elements[:-1]:
-        assert element.metadata.filename == "fake-email-attachment.msg"
-        assert element.metadata.subject == "Fake email with attachment"
-    assert elements[-1].text == "Hey this is a fake attachment!"
-    assert elements[-1].metadata == expected_metadata
-
-
-def test_partition_msg_can_process_min_max_wtih_attachments(tmp_path: pathlib.Path):
-    file_path = example_doc_path("fake-email-attachment.msg")
-    tmp_dir_path = str(tmp_path)
-    extract_msg_attachment_info(filename=file_path, output_dir=tmp_dir_path)
-    attachment_filename = str(tmp_path / str(ATTACH_EXPECTED_OUTPUT[0]["filename"]))
-
-    attachment_elements = partition_text(
-        filename=attachment_filename,
-        metadata_filename=attachment_filename,
-        min_partition=6,
-        max_partition=12,
-    )
-
+def test_partition_msg_can_process_attachments():
     elements = partition_msg(
-        filename=file_path,
-        attachment_partitioner=partition_text,
-        process_attachments=True,
-        min_partition=6,
-        max_partition=12,
-    )
-
-    assert elements[0].text.startswith("Hello!")
-    assert elements[-1].text == attachment_elements[-1].text
-    assert elements[-2].text == attachment_elements[-2].text
-    for element in elements:
-        if element.metadata.attached_to_filename is not None:
-            assert len(element.text) <= 12
-            assert len(element.text) >= 6
-
-
-def test_partition_msg_raises_with_no_partitioner():
-    with pytest.raises(ValueError):
-        partition_msg(example_doc_path("fake-email-attachment.msg"), process_attachments=True)
```

Comment:
`attachment_partitioner` parameter was removed and is now unconditionally `partition()`.